src/generators/go/presets/JsonPreset.ts

```diff
@@ -0,0 +1,17 @@
+import { FieldType, GoPreset } from '../GoPreset';
+
+/**
+ * Preset which adds basic JSON Tags for fields
+ * 
+ * @implements {GoPreset}
+ */
+export const GO_JSON_PRESET: GoPreset = {
+  struct: {
+    field({fieldName, content, type }) {
+      if (type === FieldType.field) {
+        return `${content} \`json:"${fieldName},omitempty"\``;
+      }
+      return `${content} \`json:"-"\``;
```

Comment:
I would love to see this part being tested. What happens when the type is either `additionalProperty` or `patternProperties` ? You won't add the tag?

Reply:
Correct, and this is because both of those two properties need to be unwrapped. And since the JSON tag don't support this, see https://github.com/golang/go/issues/6213, we need to customize methods instead. And this can be added later.

Reply:
But the test can be added right now, isn't?

Reply:
Yep, added!